code/ATMOSPHERICS/components/unary_devices/vent_pump.dm

```diff
@@ -263,7 +263,7 @@
 		return 1
 	if(istype(W, /obj/item/weapon/weldingtool))
 		var/obj/item/weapon/weldingtool/WT = W
-		if (WT.remove_fuel(0,user))
+		if (WT.remove_fuel(1,user))
```

Comment:
That is correct, and how I intended it, however, if you feel this is not how it should be, I can change it.